live/global.ini

```diff
@@ -65791,18 +65791,18 @@ spacecargo_illegal_analogy_0001=ein Haufen Pommes und kein Ketchup
 spacecargo_illegal_desc_0002=Ich habe erfahren, dass eine Lieferung von ~mission(Item), nach der die ~mission(Client) gesucht hat, ein ~mission(Ship) gefunden hat, das das System verlassen hat. Zum Glück kamen sie nicht zu weit. Jetzt muss nur noch jemand die Waren von der ~mission(Location) holen und sie bei ~mission(Destination) abgeben. ~mission(Timed)~mission(Danger)
 spacecargo_illegal_desc_0003=Gibt es jemanden, der sich mit Bergung auskennt und auf der Suche nach Arbeit ist? Ich habe eine Kiste mit ~mission(Item), die in der Nähe einer explodierten ~mission(Ship) darauf wartet, geborgen zu werden. ~mission(Client) braucht die Kiste bei ~mission(Destination). ~mission(Timed)~mission(Danger)Und erzählen Sie mir nicht, wie großartig Sie im EVA-Flug sind, wenn Sie keine Möglichkeit haben, selbst da rauszufliegen. Etwas Untertauchen mit Rucksack und ohne Transport nützt mir genauso viel wie ~mission(Analogy).
-spacecargo_illegal_desc_0004=Es findet ein Ausverkauf bei einer toten ~mission(Ship) statt und die ~mission(Client) sucht nach einem Läufer, der sich eine Kiste mit ~mission(Item) schnappt und sie zur ~mission(Destination) transportiert. ~mission(Timed)Dies ist der perfekte Auftritt für jemanden, der seinen Ruf entgiften möchte. ~mission(Danger)
-spacecargo_illegal_desc_0005=Eine ~mission(Ship) mit wertvoller Fracht ist explodiert und die ~mission(Client) zahlt dafür, dass diese Sendung von ~mission(Item) zur ~mission(Destination) gebracht wird. ~mission(Timed)~mission(Danger)Wenn ich höre, dass dieses Paket woanders landet und ich herausfinde, dass du etwas damit zu tun hast, dann werde ich ~mission(Threat) ausführen.
-spacecargo_illegal_desc_0006=Hey! Wer hat Lust auf Wrackerkundung? Da gibt es ein zerfetztes ~mission(Ship), aus dem eine Kiste mit ~mission(Item) geborgen und an ~mission(Destination) geliefert werden muss. Interessiert? Was ist, wenn ich dir sage, dass die ~mission(Client) die Rechnung übernehmen? ~mission(Timed) ~mission(Danger)
+spacecargo_illegal_desc_0004=In einer stillgelegten ~mission(Ship) findet gerade ein Ausverkauf statt und der ~mission(Client) sucht nach einem Zusteller, der sich eine Kiste mit ~mission(Item) schnappt und sie zur ~mission(Destination) transportiert. ~mission(Timed)Dies ist der perfekte Job für jemanden, der seinen Ruf aufpolieren möchte. ~mission(Danger)
+spacecargo_illegal_desc_0005=Eine ~mission(Ship) mit wertvoller Fracht ist explodiert und der ~mission(Client) zahlt dafür, dass die Sendung von ~mission(Item) zur ~mission(Destination) geliefert wird. ~mission(Timed)~mission(Danger)Wenn ich höre, dass dieses Paket woanders landet und ich herausfinde, dass du etwas damit zu tun hast, dann werde ich ~mission(Threat) ausführen.
```

Comment:
Satzbau passt nicht mit den threats überein

Reply:
Was passt denn nicht?

`A ~mission(Ship) with some valuable cargo blew up and the ~mission(Client) are paying to have this shipment of ~mission(Item) taken to ~mission(Destination). ~mission(Timed)~mission(Danger)If I hear that this package winds up somewhere else, and I find out you had something to do with that, then I'll ~mission(Threat).`

Reply:
Siehe https://github.com/rjcncpt/StarCitizen-Deutsch-INI/pull/404#discussion_r1444089109